svc-aggregation/agcommon/common_test.go

```diff
@@ -570,14 +552,68 @@ func mockPlugins(t *testing.T) {
 		},
 	}
 	for _, plugin := range pluginArr {
-		pl := "Plugin"
 		//Save data into Database
-		if err := connPool.Create(pl, plugin.ID, &plugin); err != nil {
+		if err := db.Create("Plugin", plugin.ID, &plugin); err != nil {
 			t.Fatalf("error: %v", err)
 		}
 	}
 }
 
+func TestGetAllPlugins(t *testing.T) {
+	config.SetUpMockConfig(t)
+	// mockDB := &MockRedisDB{}
```

Comment:
can we remove this..if not required

Reply:
Thanks for the comment, have removed it!!